observability/otel/otel-collector/otel-collector-connectors.adoc

```diff
@@ -8,6 +8,108 @@ toc::[]
 
 A connector connects two pipelines. It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It can consume and emit data of the same or different data type. It can generate and emit data to summarize the consumed data, or it can merely replicate or route data.
 
+[id="count-connector_{context}"]
+== Count Connector
+
+The Count Connector counts trace spans, trace span events, metrics, metric data points, and log records in exporter pipelines.
+
+:FeatureName: The Count Connector
+include::snippets/technology-preview.adoc[]
+
+The following are the following default metric names:
+
+* `connectors.count.trace.span.count`
+* `connectors.count.trace.span.event.count`
+* `connectors.count.metric.count`
+* `connectors.count.metric.datapoint.count`
+* `connectors.count.log.record.count`
+
+Custom metric names are also supported:
+
+* `connectors.count.spans.<custom_name_metric_for_counting_spans>`
+* `connectors.count.metrics.<custom_name_metric_for_counting_metrics>`
+* `connectors.count.logs.<custom_name_metric_for_counting_logs>
```

Comment:
These metrics are tha names you set in the configuration. I think we can remove this

Reply:
TY. Fixed.